ansible/roles/ufw/readme.md

```diff
@@ -0,0 +1,7 @@
+# UFW Module
+
+This module configures a simple firewall rule using [ufw](https://wiki.ubuntu.com/UncomplicatedFirewall).
+
+To enable it just specify a `ufw_private_interface` (e.g. `eth0`) in your environment variables.
+If you do this, inbound traffic on _all interfaces except that one_ will be blocked.
```

Comment:
This seems semi dangerous, e.g. if the interface was named differently for some reason. Can we add a rule that is something like: `ufw: rule=allow direction=in from_ip={{ ansible_control_host_ip }}`. I don't know the order of rule resolution, or what the appropriate value for `ansible_control_host_ip` would be, but this would let us recover the server.